src/components/layout/header/Header/index.tsx

```diff
@@ -91,13 +89,17 @@ const Header = () => {
             <UserMenu profileImageUrl={profileImageUrl} nickname={nickname} email={email} userMenuRef={userMenuRef} />
           </div>
         )}
+      </header>
+      <div
+        className={cx(
+          'header-drawer-menu',
+          { 'header-drawer-menu-open': isVisible },
```

Comment:
전체적으로 네스팅 문법을 사용하지만 visible 같이 요소의 표시 여부를 나타내는 동작은 아래와 같이 작성할 것 같습니다.
```suggestion
          { visible: isVisible },
          { hidden: !isVisible}
```

Reply:
반영했습니다!